project-icp/bee-pollinator-app
# Remove Micro Site/Water Balance Site

There's nothing comparable to the MMW Micro Site in the bee project, so this PR removes all reference to it.

### Testing
- Pull this branch, restart the server
- Confirm you can still navigate the app
- Go to `localhost:8000/micro` --> you should get a 404 Page Not Found
- Search through the project for reference to `water_balance` or  the micro-site (there is still a link to it in the TR55 model page, but we'll get rid of that whole page soon anyway)

Connects #18
